libcaf_io/src/instance.cpp

```diff
@@ -162,20 +162,20 @@ bool instance::handle(execution_unit* ctx, new_datagram_msg& dm,
       return err();
     }
   }
-  // Handle FIFO ordering of datagrams
```

Comment:
Why switch to lowercase? New code in CAF usually comments with whole sentences, i.e., capitalized first letter (and period at the end). All lowercase comments often indicate old code in CAF.

Reply:
I somehow thought it was the other way around. What about comments for members?

Reply:
> What about comments for members?

Same, except that they use `///` for doxygen (unless private, but even then I recommend using the doxygen style; that makes shuffling code around easier in case the access changes later).